apps/studio/src/features/dashboard/DashboardTable.tsx

```diff
@@ -25,59 +26,57 @@ import {
   BiDotsHorizontalRounded,
 } from 'react-icons/bi'
 import { MdOutlineHorizontalRule } from 'react-icons/md'
+import { trpc } from '~/utils/trpc'
 
-export const DashboardTable = (): JSX.Element => {
-  const dummyChildData: {
-    id: string
-    name: string
-    permalink: string
-    type: 'page' | 'folder'
-    status: 'folder' | 'draft' | 'published'
-    lastEditUser: string
-    lastEditDate: Date | 'folder'
-  }[] = [
-    {
-      id: '0001',
-      name: 'Test Page 1',
-      permalink: '/',
-      type: 'page',
-      status: 'draft',
-      lastEditUser: 'user1@test.com',
-      lastEditDate: new Date(),
-    },
-    {
-      id: '0003',
-      name: 'Test Folder 1',
-      permalink: '/testfolder1',
-      type: 'folder',
-      status: 'folder',
-      lastEditUser: 'folder',
-      lastEditDate: 'folder',
-    },
+// TODO: add loading state, probably not req for mvp
+
+export default function DashboardTable(): JSX.Element {
+  const router = useRouter()
+  const [pageNumber, onPageChange] = useState(1)
+  const [dataToDisplay, setDataToDisplay] = useState<
     {
-      id: '0002',
-      name: 'Test Page 2',
-      permalink: '/testpage2',
-      type: 'page',
-      status: 'published',
-      lastEditUser: 'user2@test.com',
-      lastEditDate: new Date(50000000000),
-    },
+      id: string
+      name: string
+      permalink: string
+      type: 'page' | 'folder'
+      status: 'draft' | 'published' | undefined
+      lastEditUser: string | undefined
+      lastEditDate: Date | undefined
+    }[]
+  >([])
+
+  const entriesPerPage = 6
+
+  let { siteId, resourceId } = useParams<{
+    siteId: string
+    resourceId: string
+  }>()
+
+  const { data, error, isLoading } = trpc.folder.readFolder.useQuery(
     {
-      id: '0004',
-      name: 'Test Folder 2',
-      permalink: '/testfolder2',
-      type: 'folder',
-      status: 'folder',
-      lastEditUser: 'folder',
-      lastEditDate: 'folder',
+      siteId: Number(siteId),
+      resourceId: Number(resourceId),
     },
-  ]
+    { enabled: !!resourceId },
+  )
 
-  const [pageNumber, onPageChange] = useState(1)
-  const [dataToDisplay, setDataToDisplay] = useState(dummyChildData)
+  useEffect(() => {
+    if (data) {
+      console.log(data)
+      setDataToDisplay(data.children)
+    }
+=  }, [data])
```

Comment:
[you might not need an effect](https://react.dev/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state)